artifacts/linting_rules/.gherkin-lintrc

```diff
@@ -0,0 +1,74 @@
+{
+  "no-files-without-scenarios" : "on",
+  "no-unnamed-features": "on",
+  "no-unnamed-scenarios": "on",
+  "no-dupe-scenario-names": ["on", "in-feature"],
+  "no-dupe-feature-names": "on",
+  "no-partially-commented-tag-lines": "on",
+  "indentation" : [
+    "on", {
+      "Feature": 0,
+      "Background": 2,
+      "Scenario": 2,
+      "Step": 2,
+      "Examples": 6,
+      "example": 4,
```

Comment:
```suggestion
      "example": 6,
```

Reply:
This was my comment @ravindrapalaskar17

Background: 2
example: 6

Reply:
Updated the **example** Indentation from 4 to 6 and kept the **Example** indentation at 4.